api/user/repository/remove.js

```diff
@@ -1,4 +1,4 @@
-import User from './userRepository.js';
+import {User} from '../../common/UserAndGroupRepository.js';
 
 export default async function remove(UserId) {
   await User.destroy({
```

Comment:
Add add handling of removing userGroup row after user was removed
If any record gets removed from the DB, then all linked records should be removed from UserGroupas well.

Reply:
you could add { onDelete: 'cascade' } for models in your relations.js file